projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -245,11 +245,21 @@ async function getVersion(options) {
 			({version: currentVersion} = JSON.parse(
 				await readFileAsync('package.json', 'utf8')
 			));
+
+			if (!currentVersion && fs.existsSync('lerna.json')) {
+				({version: currentVersion} = JSON.parse(
+					await readFileAsync('lerna.json', 'utf8')
+				));
```

Comment:
Not really sure if this is just me (likely is just me...), but this syntax seems hard to read. Would it make sense to update it to this?

```suggestion
				currentVersion = JSON.parse(
					await readFileAsync('lerna.json', 'utf8')
				).version;
```

Reply:
I copied it from above, but I don't like it either, unless it is prependend with a `let` or a `const`, in which case, it is more understable because you have a context for destructuring.

So (for me):

````javascript
const {version: currentVersion} = json;  :+1: 

({version: currentVersion) = json;) :-1: 
```

Do we all agree? :thinking: